packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx

```diff
@@ -130,10 +130,12 @@ function CustomViewLoader(props: TCustomViewLoaderProps) {
   ].join('/');
 
   return (
-    <CustomPanel
+    <Drawer
+      isOpen
       title={`Custom View: ${props.customView.defaultLabel}`}
       onClose={props.onClose}
-      size={panelSize}
+      // small is 10, but what is large? is it 20? or 30?
```

Comment:
Usually a property should have a set of values and the default one would be one of them. If we use `100%` here as the default value, it looks a bit weird to me as is not a value from the available set.

In other parts we use `scale` as the value for the cases when we need the `100%` space so maybe we can have something like this: `10`, `20`, `30`, `scale`.

I'd like to know what @chloe0592 thinks about it.

Reply:
@YahiaElTai first of thanks for working on it! Great job! 👏 👏 
I would also agree that it makes sense that 100% is 'scale'. And we are using that all over UI-Kit and App-Kit when setting a size to 100%, so it will be consistent.
I do however, have a question on why are we changing the `ModalPage` sizes from 't-shirt' to 'numeric'?
I mean, there is no universal standard for representing sizes but it seems to me that Drawer component in other UI libraries usually have 't-shirt' sizes since it won't be changing often and it's much more user friendly.

Reply:
I have added the new size `scale` and set it as default :)

Reply:
Thanks @chloe0592 for jumping in 👍 

Regarding this comment:
> I do however, have a question on why are we changing the ModalPage sizes from 't-shirt' to 'numeric'?
I mean, there is no universal standard for representing sizes but it seems to me that Drawer component in other UI libraries usually have 't-shirt' sizes since it won't be changing often and it's much more user friendly.

This is a due to what we discussed in our off-site about moving to numbered-range scales to be more flexible with size requirements changes.
Actually, this is something that had already started from time ago: if you take a look at dialog components (ex: [`InfoDialog`](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/application-components/src/components/dialogs/info-dialog/info-dialog.tsx#L9)) we were using t-shirt sizes in the beginning but introduced numbered-ranges later (we're keeping t-shirts for retro-compatibility for a while).

Does this make sense to you?

Reply:
Hey @CarlosCortizasCT yes, I know we talked about it, but I wasn't aware we made a decision yet. 😄 But please continue as planned. Thanks!